src/store/fs/util/entity_manager.rs

```diff
@@ -0,0 +1,1038 @@
+#![allow(dead_code)]
```

Comment:
maybe remove this?

Reply:
When I remove the `allow(dead_code)`, I don't actually get any errors for dead code right now.
Is it maybe worth reconsidering this?

Reply:
I guess it works because there's another `#![allow(dead_code)]` in the `main_actor` module.